tests/main/snap-core-symlinks/task.yaml

```diff
@@ -1,12 +1,20 @@
 summary: Check that the seed symlinks work
 
-systems: [ubuntu-core-16-*]
+systems: [ubuntu-core-1*]
 
 execute: |
-    echo "Ensure that the core snap is a symlink into the seed"
-    core_symlink="$(readlink -f /var/lib/snapd/snaps/core_*.snap)"
+    # shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB/systems.sh"
+
+    TARGET_SNAP=core
+    if is_core18_system; then
```

Comment:
(side-note) this kind of thing is exactly what I had in mind for https://github.com/snapcore/spread/issues/83, could you maybe take a look at that issue and comment on the issue if it makes sense to you?

Reply:
@anonymouse64, I think it could be really usefull, many times we are using this if ... to manage this problem. I'll create a PR to address this issue.

Thanks for reporting that.